### PR TITLE
Feature/companies page

### PR DESCRIPTION
**Pull request by winston**
Add a `companies` page to list companies in Singapore using Ruby.

Looks like this: 

screen shot 2015-10-01 at 11 56 03 pm

Fixes https://github.com/rubysg/rubysg-reboot/issues/5

### CODE REVIEW

File: app/models/company.rb [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  validates_presence_of(
+    :name,
+    :website,
+    :logo_url,
+    :address,
+    :email
+  )
+
+  def self.all
+    LIST.map { |company| Company.new(company) }.sort_by {|company| company.name.downcase }
```
Comment (Drenmi) [inline app/models/company.rb]: TIL: String sorting is affected by capitalization. :smiley:

**Comment by Drenmi** [comment]
This is great! :tada:

**Comment by winston** [comment]
Hello!

I have added this page to showcase companies in Singapore using Ruby. 
Also, if you are hiring you can put in a link to your hiring page and I hope that helps your recruitment.

*Please go ahead and add your company!*

Instructions on README: https://github.com/rubysg/rubysg-reboot#singapore-companies-using-ruby

Let me know if you think this can be further improved. Thank you!

Reason for not adding a form to add your company is because.. we are engineers! 

@rubysg/members @jaryl @Drenmi @cheeaun @teohm @miccheng @shinnyx @hanmd82  @sherwyngoh @coffeesam

Please tag more people!

**Comment by winston** [comment]
@weiqingtoh @sayanee @flyfy1 @allenlsy @marcosbeirigo @arnvald @cameronpriest @tohyongcheng

File: app/models/company.rb (hunk 1 above)
Comment (winston) [inline app/models/company.rb]: Me too. Haha Specs FTW.

**Comment by winston** [comment]
> This is great! 

Add your company! :)

**Pull request merged**